phpunit.xml.dist

```diff
@@ -16,6 +16,7 @@
     <php>
         <!-- https://github.com/symfony/console/blob/v3.3.12/Terminal.php#L24-L36 -->
         <env name="COLUMNS" value="120"/>
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="35"/>
```

Comment:
This setting allows up to 35 deprecation warnings to be triggered during test execution. I have opened #972 and #973 to address the deprecation warnings, so we can lower this to zero.

Reply:
Both PRs have been merged, so I've removed that line again.